gui/label.py

```diff
@@ -0,0 +1,18 @@
+import pygame
+
+
+class Label():
+    def __init__(self, x_pos, y_pos, text_input, font="comicsans", font_size=50, color ="yellow"):
```

Comment:
IMHO parameter `text_input` should be inside draw method. It'll be better when we dont have to import instance of player(for print HP for example) inside graphic module. When parameter `text_input` will be inside draw method instance will be imported only in one different file(Game_state).